.github/workflows/ci-schedule-compatibility.yaml

```diff
@@ -11,6 +11,7 @@ jobs:
     if: ${{ github.repository == 'karmada-io/karmada' }}
     runs-on: ubuntu-22.04
     strategy:
+      max-parallel: 5
```

Comment:
Here can add some comments for this iteration to facilitate subsequent development and understanding.

Reply:
Sure